src/content/docs/journey-migration/migrating-infra.mdx

```diff
@@ -0,0 +1,230 @@
+        <TabsPageItem id="3">
+        <SideBySide>
+        <Side>
+            <img
+            title="macOS guided install"
+            alt="An image displaying New Relic's guided installation for macOS"
+            src={journeyGuidedmacOS}
+            />
+        </Side>
+        <Side>
+        <Steps>
+            <Step>
+            Select the New Relic account you want to monitor your infrastructure data.  
+            </Step>
+            <Step>
+            Select **macOS** under the **Auto-discovery** section.
+            </Step>
+            <Step>
+            Select **Begin installation**.
+            </Step>
+        </Steps>
+        New Relic will provide a command for you to run on your macOS host. Just copy, paste, and run the command on your host, answer the prompts, and you're done!
+        </Side>
+        </SideBySide>
+        </TabsPageItem>
+        <TabsPageItem id="4">
+        <SideBySide>
+        <Side>
+            <img
+            title="Kubernetes guided install"
+            alt="An image displaying New Relic's guided installation for Kubernetes"
+            src={journeyGuidedK8}
+            />
+        </Side>
+        <Side>
+        <Steps>
+            <Step>
+            Select the New Relic account you want to monitor your infrastructure data.  
+            </Step>
+            <Step>
+            Select **Kubernetes** under the **Auto-discovery** section.
+            </Step>
+            <Step>
+            Select **Begin installation**.
+            </Step>
+        </Steps>
+        The guided install will take you through the Kubernetes implementation process. Follow the prompts and you'll be all set. If you need any help, see our [Kubernetes installation guide](/docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure/#before-start)
+        </Side>
+        </SideBySide>
+        </TabsPageItem>
+        <TabsPageItem id="5">
+        <SideBySide>
+        <Side>
+            <img
+            title="Docker guided install"
+            alt="An image displaying New Relic's guided installation for Docker"
+            src={journeyGuidedDocker}
+            />
+        </Side>
+        <Side>
+        <Steps>
+            <Step>
+            Select the New Relic account you want to monitor your infrastructure data.  
+            </Step>
+            <Step>
+            Select **Docker** under the **Auto-discovery** section.
+            </Step>
+            <Step>
+            Select **Begin installation**.
+            </Step>
+        </Steps>
+        New Relic will provide a command for you to run on your host. Copy, paste, and run the command, and you're done! 
+        </Side>
+        </SideBySide>
+        </TabsPageItem>
+    </TabsPages>
+</Tabs>
+
+New Relic also offers a whole suite of on-host integrations. From **Add Data > Infrastructure & OS**, just select your desired integration from our list to get started. 
```

Comment:
The UI path here should include the product URL as a starting point https://docs.newrelic.com/docs/style-guide/formatting/ui-paths/

Reply:
Fixed (I think)